aptos-move/framework/aptos-token/sources/bid.move

```diff
@@ -0,0 +1,281 @@
+module aptos_token::bid {
```

Comment:
would be good to document the flow at the top here.

Reply:
add doc pointing to the example